frontend/apps/reader/modules/readerview.lua

```diff
@@ -782,8 +778,7 @@ function ReaderView:onReadSettings(config)
     self.page_scroll = page_scroll == 1 and true or false
     self.highlight.saved = config:readSetting("highlight") or {}
     self.page_overlap_style = config:readSetting("page_overlap_style") or G_reader_settings:readSetting("page_overlap_style") or "dim"
-    self.page_gap.height = config:readSetting("kopt_page_gap") or 8
-    self.page_gap_width = config:readSetting("kopt_page_gap") or 8
+    self.page_gap.height = Screen:scaleBySize(config:readSetting("kopt_page_gap_height") or 8)
```

Comment:
you might want
```diff
-   self.page_gap.height = Screen:scaleBySize(config:readSetting("kopt_page_gap_height") or 8)
+   self.page_gap.height = Screen:scaleBySize(config:readSetting("kopt_page_gap_height") or G_reader_settings:readSetting("page_gap_width") or 8)
```

Reply:
that should be 
`self.page_gap.height = Screen:scaleBySize(config:readSetting("kopt_page_gap_height") or G_reader_settings:readSetting("kopt_page_gap_hight") or 8)`

also this does not work with defaults (holding the option) with or without this change

Reply:
sorry typo caused that.
to get defaults to work you need this
```diff
-    self.page_gap.height = Screen:scaleBySize(config:readSetting("kopt_page_gap_height") or 8)
+    self.page_gap.height = Screen:scaleBySize(config:readSetting("kopt_page_gap_height") or G_reader_settings:readSetting("kopt_page_gap_height") or 8)
```